### PR TITLE
Update Lab 3

### PR DESCRIPTION
**Pull request by madchr1st**
Rewrite lab3 for baloise

### CODE REVIEW

File: content/en/docs/03/_index.md [hunk 1]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
 Alertmanager's configuration is done using a YAML config file and CLI flags. Take a look at the very basic configuration file at `/etc/alertmanager/alertmanager.yml`:
 
+{{% /onlyWhenNot %}}
+{{% onlyWhen baloise %}}
+
+At Baloise the Alertmanager is part of the managed monitoring stack and does not need to be installed. We will have a look at the default configuration in the next chapter.
+
+### Configuration
+
+Alertmanager's configuration is managed by the monitoring stack and can be configured done using a YAML config file and CLI flags. Take a look at the default configuration in use at Baloise:
```
Comment (mebagel) [inline content/en/docs/03/_index.md:44]:
```suggestion
Alertmanager's configuration is managed by the monitoring stack and can be configured using a YAML config file and CLI flags. Take a look at the default configuration in use at Baloise:
```

File: content/en/docs/03/labs/baloise_alertmanager-tasks.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+```
+rate(kube_pod_container_status_restarts_total{job="kube-state-metrics",namespace="testnamespace"}[5m]) * 60 * 5 > 0
+```
+
+You can build/verify your Query in your Thanos Querier UI. As soon, as you apply the PrometheusRule resource, you should be able to see the alert in your Thanos Ruler implementation.
+
+### Task {{% param sectionnumber %}}.2: Send a test alert
+
+In this taks you can use the [amtool](https://github.com/prometheus/alertmanager#amtool) command to send a test alert.
```
Comment (mebagel) [inline content/en/docs/03/labs/baloise_alertmanager-tasks.md:59]:
```suggestion
In this task you can use the [amtool](https://github.com/prometheus/alertmanager#amtool) command to send a test alert.
```

File: content/en/docs/03/_index.md [hunk 3]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+
+```yaml
+# baloise config
+global:
+  resolve_timeout: 5m
+  http_config:
+    follow_redirects: true
+  smtp_from: devops@baloise.com
+  smtp_hello: localhost
+  smtp_smarthost: smtp.baloisenet.com:25
```
[the remark this answers is not in the document]
Comment (madchr1st) [reply content/en/docs/03/_index.md:54]: Why? It is only used to show the relatively complex configuration at Baloise.

File: content/en/docs/03/_index.md [hunk 4]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+
+Alertmanager's configuration is managed by the monitoring stack and can be configured done using a YAML config file and CLI flags. Take a look at the default configuration in use at Baloise:
+
+```yaml
+# baloise config
+global:
+  resolve_timeout: 5m
+  http_config:
+    follow_redirects: true
+  smtp_from: devops@baloise.com
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:52]:
```suggestion
  smtp_from: devops@example.com
```

File: content/en/docs/03/_index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (madchr1st) [reply content/en/docs/03/_index.md:54]: Gotcha

Comment (madchr1st) [inline content/en/docs/03/_index.md:54]:
```suggestion
  smtp_smarthost: smtp.example.com:25
```

File: content/en/docs/03/_index.md [hunk 5]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+    severity: info
+  equal:
+  - namespace
+  - alertname
+receivers:
+- name: default
+- name: mail-critical
+  email_configs:
+  - send_resolved: false
+    to: group.ch_dcs_devops_system@baloise.ch
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:135]:
```suggestion
    to: group.devops_system@example.com
```

File: content/en/docs/03/_index.md [hunk 6]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+    from: devops@baloise.com
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:136]:
```suggestion
    from: devops@example.com
```

File: content/en/docs/03/_index.md [hunk 7]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+    hello: localhost
+    smarthost: smtp.baloisenet.com:25
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:138]:
```suggestion
    smarthost: smtp.example.com:25
```

File: content/en/docs/03/_index.md [hunk 8]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+    headers:
+      From: devops@baloise.com
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:140]:
```suggestion
      From: devops@example.com
```

File: content/en/docs/03/_index.md [hunk 9]
```diff
@@ -33,6 +34,169 @@ The configuration file of alertmanager is located here: `/etc/alertmanager/alert
...
+      Subject: '{{ template "email.default.subject" . }}'
+      To: group.ch_dcs_devops_system@baloise.ch
```
Comment (madchr1st) [inline content/en/docs/03/_index.md:142]:
```suggestion
      To: group.devops_system@example.com
```

**Review by mebagel — APPROVED** [review]

**Pull request merged**